lightkurve/periodogram.py

```diff
@@ -371,6 +508,40 @@ def flatten(self, method='logmedian', filter_width=0.01, return_trend=False):
             return snr, bkg
         return snr
 
+    def _get_fwhm(self, numax):
+        """For a given numax, the method calculates the expected Full Width Half
+        Maximum of the seismic mode envelope as (Mosser et al 2010) for RGB
+        stars:
+
+        fwhm = 0.66 * numax^0.88,
+
+        if the data is generated for a long-cadence timeseries. If the
+        timeseries is short-cadence, the relation will be:
+
+        fwhm = 0.25 * numax.
+
+        This will be based on the highest frequency in the periodogram.
+
+        Parameters:
+        ----------
+            numax : float
+                The estimated position of the numax of the power spectrum. This
+                is used to calculated the region autocorrelated with itself.
+
+        Returns:
+        --------
+            fwhm: float
+                The estimate full-width-half-maximum of the seismic mode envelope
+        """
+        fs = np.median(np.diff(self.frequency.value))
```

Comment:
`fs` doesn't seem to be used in the remainder of this method, so this line can probably be removed!